manifests/init.pp

```diff
@@ -53,6 +55,7 @@
   String $server_store_package = $::qpid::params::server_store_package,
   Boolean $ssl = $::qpid::params::ssl,
   Integer[0, 65535] $ssl_port = $::qpid::params::ssl_port,
+  Integer $session_unacked = $::qpid::params::session_unacked,
```

Comment:
You could use ```Integer[0]``` to set a lower bound of 0, assuming negative integers are not supported.